src/write-compressed-ply.ts

```diff
@@ -0,0 +1,110 @@
+import { FileHandle } from 'node:fs/promises';
```

Comment:
Thinking it might be cool to have writers and readers folders and make it simple for us (or others) to add support for other formats.